src/makeFetchAction.js

```diff
@@ -1,5 +1,28 @@
 import get from './utils/get';
-import { CALL_API, REDUCER_PATH, ACTION_UPDATE_LOCAL } from './constants';
+import {
+  CALL_API,
+  REDUCER_PATH,
+  ACTION_UPDATE_LOCAL,
+  ACTION_RESET_LOCAL,
+} from './constants';
+
+const normalizeResetData = data => {
```

Comment:
oh, so you did it here, but there is no test. Let's use the default parameter syntax instead and move defaultData to the outer scope. Maybe we should set it as default params for `resetter` itself (line 43)